content/components/icons.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: "Icons"
+description: "Use [Font Awesome](https://fontawesome.com/icons) icons anywhere in the doc"
+icon: "icons"
+---
+
+The documentation <Icon icon="book-open" /> you want, effortlessly
```

Comment:
I think we should add a large icon in it's own line

Reply:
You can also specify later that it can either be on it's own line or inline

Reply:
np,  @handotdev  how about
![image](https://github.com/mintlify/docs/assets/16908811/0371dcaf-f86d-457d-a55a-a71e3f4fb5fa)